libs/dyn/convert/from_typed.go

```diff
@@ -126,7 +126,9 @@ func fromTypedStruct(src reflect.Value, ref dyn.Value, options ...fromTypedOptio
 
 		// Either if the key was set in the reference or the field is not zero-valued, we include it.
 		if ok || nv.Kind() != dyn.KindNil {
-			out.Set(refk, nv)
+			if err := out.Set(refk, nv); err != nil {
+				return dyn.InvalidValue, err
```

Comment:
I think it might be safer to log the error here and not return the error. We might even intentionally swallow the error here, cc @pietern as you might know more

Reply:
I'm okay with that. @pietern what is your opinion?

Reply:
I've reverted all changes to libs/dyn and annotated the cases there with //nolint:errcheck